examples/dev_flow/debug/debugging_visual.rst

```diff
@@ -0,0 +1,72 @@
+
+.. _examples_dev_flow_debug_visual:
+
+
+Debugging shared libraries with Visual Studio
```

Comment:
Please, check other examples in the examples section of the docs, we usually add code for the example, in the examples2 repo: https://github.com/conan-io/examples2/
Maybe we can consider if it would make sense to add one, with some screenshots and also it would be nice to invite the user to open a project in visual studio, then using conan cache clean to clean all build folders and showing them that with the hook even after doing that you can still debug the libraries (also remark the difference with the previous section of the docs on that matter: https://docs.conan.io/2/examples/dev_flow/debug/step_into_dependencies.html )